compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -105,14 +103,15 @@ fun TooltipArea(
     content: @Composable () -> Unit
 ) {
     var parentBounds by remember { mutableStateOf(Rect.Zero) }
-    var popupPosition by remember { mutableStateOf(Offset.Zero) }
     var cursorPosition by remember { mutableStateOf(Offset.Zero) }
     var isVisible by remember { mutableStateOf(false) }
     val scope = rememberCoroutineScope()
     var job: Job? by remember { mutableStateOf(null) }
 
     fun startShowing() {
-        job?.cancel()
+        if (job?.isActive == true) {  // Don't restart the job if it's already active
+            return
+        }
```

Comment:
This is needed because this function is now potentially called on every mouse-move, and we don't want to restart the timer.